moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js

```diff
@@ -87,6 +88,46 @@ export const ProjectsListViewQueryConf = {
         canceled: "default",
       },
     },
+    current_phase: {
+      searchable: true,
+      sortable: false,
+      label: "Current Phase",
+      width: "15%",
+      search: {
+        label: "Search by current phase",
+        operator: "_ilike",
+        quoted: true,
+        envelope: "%{VALUE}%",
+      },
+      type: "string",
+    },
```

Comment:
Maybe just one last observation, the phase is lower-case where we should probably try to capitalize the string.

Reply:
found out `"text-transform": "capitalize",` was there in the class, but wasn't being used. its capitalized now!